lib/models/ColdStorageActionQueueEntry.js

```diff
@@ -0,0 +1,68 @@
+const joi = require('joi');
+
+const entrySchema = joi.object({
+    op: joi.string().valid('restore', 'delete', 'archive').required(),
+    bucketName: joi.string().required(),
+    objectKey: joi.string().required(),
+    objectVersion: joi.string().required(),
+    accountId: joi.string().required(),
+    archiveInfo: joi.object().required(),
+    requestId: joi.string().required(),
+});
+
+class ColdStorageActionQueueEntry {
```

Comment:
This seems to describe the result of an action rather than an action, which was confusing me.

Reply:
will rename to be more specific to cold storage status as opposed to trying to be a general cold storage name